### DISCUSSION TITLE
Want to change icon from PaletteView

### DISCUSSION THREAD

**Discussion by carlosmreyes** [eclipse-gef/gef-classic#262] [Q&A] [answered]
Hi, we have an application that uses an editor that extends from "GraphicalEditorWithFlyoutPalette". In this editor, there is a palette and we want to change the icon from the top of it (see screenshot).
![image](https://github.com/eclipse/gef-classic/assets/104150032/fb591cb5-f294-4133-8b8f-756719378c56)

We were checking the source code of "gef-classic" project and we noticed that the icon is defined in a view in this way:
`<extension
			point="org.eclipse.ui.views">
		<view
				name="%Palette.Label"
				icon="icons/palette_view.gif"
				category="org.eclipse.ui"
				class="org.eclipse.gef.ui.views.palette.PaletteView"
				id="org.eclipse.gef.ui.palette_view">
		</view>
	</extension>`

Do you know if it's possible to use another icon in our application?

Thanks in advance.

**Comment by Destrolaric** [comment, marked as answer]
@carlosmreyes 
`
InternalImages.set(InternalImages.IMG_PALETTE, <your image>);
`

**Reply by carlosmreyes** [reply]
It works :) 
Thanks a lot, @Destrolaric
